src/cli/services/linter/linter.ts

```diff
@@ -91,5 +91,20 @@ export async function lint(documents: Array<number | string>, flags: ILintConfig
     );
   }
 
+  filesNotFound.forEach(fileNotFound => {
```

Comment:
Minor style issue here.
We use `for of` in most places.
```suggestion
  for (const fileNotFound of filesNotFound) {
```